<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-RKOMUFV" failures="0" tests="2" name="com.day12.facebookPara.com.faceBookLoginPage" time="8.302" errors="1" timestamp="17 Aug 2021 10:08:54 GMT">
  <testcase classname="com.day12.facebookPara.com.faceBookLoginPage" name="initilizeWebEnvirnment" time="8.302"/>
  <testcase classname="com.day12.facebookPara.com.faceBookLoginPage" name="loginUsingCredentials" time="0.000">
    <error message="Can&#039;t invoke method public java.lang.Object[][] com.day12.facebookPara.com.TestData.getData() on this instance of class com.day12.facebookPara.com.faceBookLoginPage due to class mismatch" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: Can't invoke method public java.lang.Object[][] com.day12.facebookPara.com.TestData.getData() on this instance of class com.day12.facebookPara.com.faceBookLoginPage due to class mismatch
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:75)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1274)
	at org.testng.internal.Invoker.createParameters(Invoker.java:989)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1079)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- loginUsingCredentials -->
</testsuite> <!-- com.day12.facebookPara.com.faceBookLoginPage -->
